configs/train/acdc_seg/adapt_pre_trained_without_decoder_ct/train_config.yaml

```diff
@@ -49,7 +49,7 @@ net:
         in_channels: 1
         out_channels: 1
         weight_path: './weights/models_genesis.pth'
-        with_decoder: false
+        loaded_modules: ['encoder']
```

Comment:
Is the syntax of the sequence in the `yaml`
```
    loaded_modules:
        - 'encoder'
        - 'decoder'
```
or
```
     loaded_modules: ['encoder', 'decoder']
```

Reply:
The types of `loaded_modules` are all `BoxList`, so I think they are the same.

Reply:
OK.